app/config/locale/translations/od.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'settings.inspire' => '"ଜ୍ଞାନୀ ହେବାର କଳା ହେଉଛି କ’ଣ ଅଣଦେଖା କରାଯିବ ଜାଣିବାର କଳା |"', // This is the line printed in the homepage and console 'view-source'
+    'settings.locale' => 'od',
+    'settings.direction' => 'ltr',
+
+    // Service - Users
+    'account.emails.team' => '%s ଟିମ',
+    'account.emails.verification.title' => 'ଏକାଉଣ୍ଟ ଭେରିଫିକେସନ',
+    'account.emails.verification.body' => 'en.email.auth.confirm.tpl',
```

Comment:
you should change en.email.auth.confirm.tpl =>od.email.auth.confirm.tpl
similar with bellow all replace en. with your local language ISO code 

best,
spielers